### DISCUSSION TITLE
[Question] Performance reports

### DISCUSSION THREAD

**Discussion by cyborgdennett** [0xPolygon/polygon-edge#962] [General]
# [ Amount of machines used in performance report ]
## Description
In the performance reports, I see under Host Configuration: c6a.48large. This is a 192 core machine. [link](https://wiki.polygon.technology/docs/edge/performance-reports/test-history/test-2022-07-04)
My question is as follows: Do each of the 4 Nodes use a single 192 core machine, or are the 4 nodes running on the same 192 core machine?

So answers are:
A. Single machine
B. Four machines

yours,

**Comment by laviniat1996** [comment]
Hi @cyborgdennett ,

The correct answer is B. Each one of those four nodes had 192 cores and was c6a.48large.

**Reply by cyborgdennett** [reply]
Thanks for the fast response!

**Reply by laviniat1996** [reply]
You are very welcome. Feel free to reach out anytime you need.
